CHANGELOG.md

```diff
@@ -1,12 +1,38 @@
+
 # nf-core/methylseq
 
-## v1.6dev - [date]
+## v1.6dev - [2020-02-15]
 
 ### Pipeline Updates
 
 * Updated template to tools 1.11
 * Moved parameter documentation into new `nextflow_schema.json` file.
 * Added new `--maxins` and `--minins` parameters to pass on to Bismark
+* Added Picard CollectInsertSizeMetrics and Picard CollectGcBiasMetrics
+* Improved qulimap and preseq by adding `samtools sort` and `samtools index` step in the Bismark aligner
```

Comment:
I have a vague memory of this being a bad idea for some reason, that Bismark alignments _shouldn't_ be sorted. But it's from years ago and I can't remember the reasoning now..

@FelixKrueger do you know what I'm talking about?

Reply:
Yes, bismark_deduplication needs an unsorted bam, so it gets the unsorted bam